apps/balancer-tools/src/app/apr/pool/(components)/PoolOverviewCards.tsx

```diff
@@ -38,7 +42,11 @@ export default async function PoolOverviewCards({
         { title: "TVL", content: formatNumber(tvl) },
         { title: "APR", content: formatNumber(apr).concat("%") },
         { title: "Round Number", content: roundId },
-        { title: "Round Ended", content: formatDate(round.endDate) },
+        {
+          title: "Round Ended",
```

Comment:
Think we should call this `veBAL Round End Date`

Reply:
nevermind, your comment wasn't showing for me hehehhe